pages/linux/wipefs.md

```diff
@@ -7,10 +7,14 @@
 
 `sudo wipefs {{/dev/sdX}}`
 
-- Wipe all available signatures for specified device:
+- Wipe all available signature types for the device (no recursion into partitions):
```

Comment:
**fixup:** When referring to a single entity such as a file or a device, tldr pages use "a specific {{x}}" by convention:

```suggestion
- Wipe all available signature types for a specific device (no recursion into partitions):
```

**Question:** Regarding the change from 'signatures' to 'signature types', I don't understand this change; I would expect that wiping a signature type indicates that we're somehow erasing the concept of a given signature (gpt, mbr etc), and wiping signatures would actually remove the signatures on-disk. Looking at the [wipefs 2.37.85-637cc man page](https://www.man7.org/linux/man-pages/man8/wipefs.8.html) shows that the term "signature type" isn't really used.

Is there something that I'm understanding that makes "signature type" more correct in this context?

Reply:
> Is there something that I'm understanding that makes "signature type" more correct in this context?

Hi @adrienthebo,  it's a fair question. The reason I made it more verbose from just "all available signatures" to all "available signature types" was that the `--all` means erasing all signatures of any type at the specified block level, but not all, as in recusing into all/deeper block levels and it does not erase "sub-level" block device signatures. I can see how this is unclear. Perhaps it should be worded:

"Wipe all signatures of any type at a specified device block layer (no recursion into partitions)". Does this sound better? Or would you prefer sticking to the original wording given "(no recursion into partitions)" does make it clear enough?

Maybe the man page helps:

> -a, --all
> Erase all available signatures. The set of erased signatures can be restricted with the -t option.

> -t, --types list
> Limit  the  set of printed or erased signatures.  More than one type may be specified in a comma-separated list. The list or individual types can be prefixed with 'no' to specify the types on which no action should be taken.  For more details see mount(8).

My main issue was that I confused all to imply nested/recursive cleanup, but it doesn't. I noticed other people in various forums shared my misconception.

Reply:
Seeing no further discussion about why or if the signature type matters, I'll leave this as be and resolve the conversation. It it still seems disagreeable, please open a fresh comment and I'll happily take any suggestions on cleaner wording.